src/Kernel/Process.class.st

```diff
@@ -39,14 +39,6 @@ Class {
 	#category : #'Kernel-Processes'
 }
 
-{ #category : #'process specific' }
-Process class >> psKeysSema [
-	"Isolate handling of class variable"
-
-	PSKeysSema ifNil: [ PSKeysSema := Semaphore forMutualExclusion ].
-	^PSKeysSema
-]
-
```

Comment:
This is super strange, I don't understand why this method got moved down...
Actually, my commit is sorting it. So somebody commited non-sorted methods?

Reply:
I saw this code in recent PRs but I do not remember which one.

Reply:
https://github.com/pharo-project/pharo/pull/4106/files